contrib/drivers/filesharedrivers/nfs/cli.go

```diff
@@ -160,6 +160,21 @@ func (c *Cli) CreateFileShare(lvPath string) error {
 	return err
 }
 
+func (c *Cli) CreateFileShareFromSnapshot(lvPath string) error {
+	cmd := []string{
+		"env", "LC_ALL=C",
+		"lvconvert",
```

Comment:
Should we specify the full path of the command? a general good practise. I understand that we have followed this approach wherever but it may help for scenarios of path change after mounting

Reply:
I didn't got the actual question(may be i will discuss offline). But in addition i can say, this is linux command and it is independent of path changes.

Reply:
Agree. Its a good practice to add full path before hitting the command. Thanks for pointing out

Reply:
Should we not set the environment setup needed for the same and issue the cmd? (instead of full path with each command?) Please check

Reply:
Agree if the environment is set earlier then full command path may not be required. 
As long as $PATH is set even for non-default location it will work.
One more advantage i see is for debugging. You know which location binary is used while debug